packages/fxa-settings/src/models/integrations/base-integration.ts

```diff
@@ -95,11 +96,6 @@ export abstract class Integration<
     this.features = { ...this.features, ...features } as T;
   }
 
-  // TODO: do we need this/isSync?
```

Comment:
The reason I removed this method and left `isSync` is because:
* Our OAuth integration has methods on it that other integrations do not have. Because of this, the `isOAuthIntegration` type guard is needed unless we want to add all of these methods to the base integration. I tried turning `.isOAuth()` into a type guard but that ended up getting really tricky for our partial interfaces and it's not a pattern I saw us use elsewhere. If we need to expand `OAuthIntegration` in the future, I'm thinking we use a type guard that accounts for both types, e.g. we do something like we change the name of `OAuthIntegration` to `OAuthRedirectIntegration` and let `isOAuthIntegration(integration)` account for all types of OAuth integrations
* Different kinds of integrations can be sync, but we don't have Sync-specific methods that warrant new integrations for these, so `isSync()` seems preferable

Reply:
It might make sense to after we can get rid of desktop v3, then we could have a single sync integration extend the OAuth integration.